flintrock/services.py

```diff
@@ -263,6 +267,15 @@ def install(
                     """.format(
                         repo=shlex.quote(self.git_repository),
                         commit=shlex.quote(self.git_commit)))
+            ssh_check_output(
+                client=ssh_client,
+                command="""
+                    set -e
+                    for f in $(find spark/bin -type f -executable ! -name '*.cmd'); do
+                        sudo ln -s $(pwd)/$f /usr/local/bin/$(basename $f)
+                    done
+                    echo -e "export SPARK_HOME=$(pwd)/spark" >> .bashrc
```

Comment:
I like the `.bashrc` version better. Can we quote the value of `SPARK_HOME` though?

i.e. Something like this:

```sh
echo "export SPARK_HOME='$(pwd)/spark'" >> .bashrc
```

That will protect against unintentional whitespace breaks.